aws/resource_aws_lambda_event_source_mapping.go

```diff
@@ -299,6 +311,9 @@ func resourceAwsLambdaEventSourceMappingRead(d *schema.ResourceData, meta interf
 	if err := d.Set("destination_config", flattenLambdaEventSourceMappingDestinationConfig(eventSourceMappingConfiguration.DestinationConfig)); err != nil {
 		return fmt.Errorf("error setting destination_config: %s", err)
 	}
+	if err := d.Set("topics", flattenStringSet(eventSourceMappingConfiguration.Topics)); err != nil {
+		return fmt.Errorf("error setting topics: %s", err)
```

Comment:
use %w

Reply:
#12991.

Reply:
I'll leave so as not to have to re-run the long acceptance test 😄.
I assume there will be a one-time bulk update of all such occurences.